language/types_values_variables.md

```diff
@@ -223,7 +236,7 @@ directly to the puppet type system.
 
 ### Ruby Object to Type Mapping
 
-The Ruby implementation of the Puppet Programming Language uses the Ruby classes `String`, `Integer` (`Fixnum`, `Bignum`, etc), `Float`, `Hash`, `Array`, `Regexp`, `TrueClass`, `FalseClass`, `NilObj`. Instances of these Ruby types are directly mapped to the corresponding puppet types (e.g. even if an instance of a puppet `String` is an instance of the Ruby class called `String`, it is not interpreted as `Ruby[String]`.
+The Ruby implementation of the Puppet Programming Language uses the Ruby classes `String`, `Integer` (`Fixnum`, `Bignum`, etc), `Float`, `Hash`, `Array`, `Regexp`, `TrueClass`, `FalseClass`, `NilObj`. Instances of these Ruby types are directly mapped to the corresponding puppet types (e.g. even if an instance of a puppet `String` is an instance of the Ruby class called `String`, it is not interpreted as `Runtime['Ruby', 'String']`.
```

Comment:
Perhaps remove (`Fixnum`, `Bignum`, etc) here since thery are no longer used >= Ruby 2.4. The meaning of "Ruby classes" is clear anyway.